<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="3,1,1">
   <item row="1" column="0">
    <widget class="QGroupBox" name="options_gb">
     <property name="title">
      <string>Packages</string>
     </property>
     <layout class="QGridLayout" name="options_grid" rowminimumheight="0"/>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="extras_gb">
     <property name="title">
      <string>Devtools and Extras</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="project_group">
     <property name="title">
      <string>Project Setup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Project Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Python Version</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="simple_script">
           <property name="text">
            <string>simple script</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="dry_run">
           <property name="text">
            <string>dry run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_script">
           <property name="text">
            <string>save script</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="create_project">
           <property name="text">
            <string>create project</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="project_location">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Project Template</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="template_choice"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="which_python"/>
      </item>
      <item row="2" column="1">
       <widget class="QPlainTextEdit" name="description_text">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="select_location">
        <property name="text">
         <string>Location</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="project_name">
        <property name="text">
         <string>MyProject</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="checkBox_3">
           <property name="text">
            <string>no readme</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>use git</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>no-workspace</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
